optimum/exporters/openvino/convert.py

```diff
@@ -486,3 +487,49 @@ def export_models(
 
     outputs = list(map(list, zip(*outputs)))
     return outputs
+
+
+OV_TOKENIZER_FILE_NAME = "openvino_tokenizer{}.xml"
+OV_DETOKENIZER_FILE_NAME = "openvino_detokenizer{}.xml"
+UNSUPPORTED_TOKENZIER_CLASSES = (
+    T5Tokenizer,
+    T5TokenizerFast,
+)
+
+
+def export_tokenizer(
+    tokenizer,
+    output_path: Union[str, Path],
```

Comment:
to keep consistency we can change to
```suggestion
    output: Union[str, Path],
```